diff --git a/node_modules/@c8y/client/lib/src/core/IFetchResponse.d.ts b/node_modules/@c8y/client/lib/src/core/IFetchResponse.d.ts
index 99fbaba..e97d183 100644
--- a/node_modules/@c8y/client/lib/src/core/IFetchResponse.d.ts
+++ b/node_modules/@c8y/client/lib/src/core/IFetchResponse.d.ts
@@ -13,5 +13,5 @@ export interface IFetchResponse {
     /**
      * Data in json format
      */
-    json(): any;
+    json(): Promise<any>;
 }
diff --git a/node_modules/@c8y/client/lib/src/core/ObservableList.d.ts b/node_modules/@c8y/client/lib/src/core/ObservableList.d.ts
index eeb0e26..94df0f6 100644
--- a/node_modules/@c8y/client/lib/src/core/ObservableList.d.ts
+++ b/node_modules/@c8y/client/lib/src/core/ObservableList.d.ts
@@ -3,7 +3,7 @@ import { IResultBase } from './IResultBase';
 /**
  * Creates an observable list (or detail) from a promise-callback.
  */
-export declare class ObservableList<TData> extends Observable<TData> {
+export declare class ObservableList<TData> extends Observable<TData[]> {
     private realtimeFilter;
     private options;
     private seed;
