diff --git a/node_modules/@c8y/ngx-components/fesm5/c8y-ngx-components.js b/node_modules/@c8y/ngx-components/fesm5/c8y-ngx-components.js
index 288a098..4303b6f 100644
--- a/node_modules/@c8y/ngx-components/fesm5/c8y-ngx-components.js
+++ b/node_modules/@c8y/ngx-components/fesm5/c8y-ngx-components.js
@@ -9021,6 +9021,7 @@ var NavigatorNodeComponent = /** @class */ (function () {
          * @return {?}
          */
         function () {
+            _this.open = _this.open || !!_this.node.open;
             _this.updateIcon();
             _this.expandParentIfActive();
         }));
